_posts/2022-04-01-release-kubernetes-operator-0.1.0.md

```diff
@@ -0,0 +1,110 @@
+
+**Core Features**
+
+ * Deploy and monitor Flink Application and Session deployments
+ * Upgrade, suspend and delete Flink deployments
+ * Full logging and metrics integration
+
+<div style="line-height:60%;">
+    <br>
+</div>
+
+<center>
+	<figure>
+	<img src="{{ site.baseurl }}/img/blog/2022-04-01-release-kubernetes-operator-0.1.0/overview.svg" width="600px" alt="Overview 1"/>
+	<br/><br/>
+	</figure>
+</center>
+
+<div style="line-height:150%;">
+    <br>
+</div>
+
+## Getting started
+
+For a detailed [getting started guide](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-release-0.1/docs/try-flink-kubernetes-operator/quick-start/) please check the documentation site.
+
+## FlinkDeployment CR overview
+
+When using the operator, users create `FlinkDeployment` objects to describe their Flink application and session clusters deployments.
+
+A minimal application deployment yaml would look like this:
+
+```yaml
+apiVersion: flink.apache.org/v1alpha1
+kind: FlinkDeployment
+metadata:
+  namespace: default
+  name: basic-example
+spec:
+  image: flink:1.14
+  flinkVersion: v1_14
+  flinkConfiguration:
+    taskmanager.numberOfTaskSlots: "2"
+  serviceAccount: flink
+  jobManager:
+    replicas: 1
+    resource:
+      memory: "2048m"
+      cpu: 1
+  taskManager:
+    resource:
+      memory: "2048m"
+      cpu: 1
+  job:
+    jarURI: local:///opt/flink/examples/streaming/StateMachineExample.jar
+    parallelism: 2
+    upgradeMode: stateless
+```
+
+Once applied to the cluster using `kubectl apply -f your-deployment.yaml` the operator will spin up the application cluster for you.
+If you would like to upgrade or make changes to your application, you can simply modify the yaml and submit it again, the operator will execute the necessary steps (savepoint, shutdown, redeploy etc.) to upgrade your application.
+
+To stop and delete your application cluster you can simply call `kubectl delete -f your-deployment.yaml`.
+
+You can read more about the [job management features](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-release-0.1/docs/custom-resource/job-management/) on the documentation site.
+
+## Release Resources
+
+The source artifacts and helm chart are now available on the updated [Downloads](https://flink.apache.org/downloads.html)
+page of the Flink website.
+
+The [official 0.1.0 release archive](https://archive.apache.org/dist/flink/flink-kubernetes-operator-0.1.0) doubles as a Helm repository that you can easily register locally:
+
+{% highlight bash%}
+$ helm repo add flink-kubernetes-operator-0.1.0 https://archive.apache.org/dist/\
+flink/flink-kubernetes-operator-0.1.0
+$ helm install flink-operator flink-kubernetes-operator-0.1.0/flink-operator --set webhook.create=false
+{% endhighlight%}
+
+You can also find official Kubernetes Operator Docker images of the new version on [Dockerhub](https://hub.docker.com/r/apache/flink-kubernetes-opeartor).
```

Comment:
Typo here, I think it should be: https://hub.docker.com/r/apache/flink-kubernetes-operator?

Reply:
good catch,thank you